src/core/Table/TableRowMemoized.tsx

```diff
@@ -66,5 +66,8 @@ export const TableRowMemoized = React.memo(
     prevProp.onBottomReached === nextProp.onBottomReached &&
     prevProp.row.original === nextProp.row.original &&
     prevProp.state.selectedRowIds?.[prevProp.row.id] ===
-      nextProp.state.selectedRowIds?.[nextProp.row.id],
+      nextProp.state.selectedRowIds?.[nextProp.row.id] &&
+    prevProp.row.cells.every(
+      (cell, index) => nextProp.row.cells[index].column === cell.column,
```

Comment:
I was literally looking at this comparator just now, and suddenly your PR shows up 😄

Reply:
lol 😀 Great that you haven't created a PR with the same changes 😀